src/ui/editor.rs

```diff
@@ -25,6 +25,8 @@ pub enum Editor {
     Emacsclient,
     Hx,
     Helix,
+    St,
```

Comment:
I believe `Subl` would be more appropriate as an alternative for full editor name